README.md

```diff
@@ -0,0 +1,27 @@
+# Examining Batch Effects on Cell Painting Data
+
+This project examines the presence of batch effects in (high-throughput) single-cell morphology data of different strains of cells subjected to various perturbations.
+In addition to evaluating the benefits of the kBET detection algorithm, Harmony is used to correct for the observed batch effect. Ultimately, the goal is to benchmark these two processes and evaluate their effectiveness on the LINCS cell morphology data.
+
+@TODO update README
+
```

Comment:
```suggestion
```

No need for this, let's just delete

Reply:
Oops, I meant to delete that, just wanted to see is annotations were a thing in Github like in Java.